deploy/helm/sumologic/values.yaml

```diff
@@ -463,6 +484,10 @@ sumologic:
 
     sourceType: otlp
 
+    ## additional exporters which may be used in any of the pipeline
+    ## remember add them to containers, systemd or kubelet routing as well
+    extraExporters: {}
```

Comment:
I will look at that. Unification may be not so trivial AFAIR

Reply:
It may be breaking change, as we have configuration per pipeline, especially `extraProcessors` and `sources` configuration, but I can implement one routing processor for all signals 👍

Reply:
Due to https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3711#discussion_r1606381354 we cannot implement this feature for http source. As we cannot have one default for all three pipelines